static/src/javascripts/lib/config.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import pad from 'lib/pad';
```

Comment:
☹️ no it won't

> NOTE: Instance methods such as "foobar".includes("foo") will not work since that would require modification of existing builtins (Use babel-polyfill for that).

https://babeljs.io/docs/plugins/transform-runtime/